pkgs/by-name/av/avdump3/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, stdenv, writeScriptBin, fetchzip, dotnet-runtime, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "avdump3";
+  version = "8293_stable";
+
+  src = fetchzip {
+    url = "https://cdn.anidb.net/client/avdump3/avdump3_8293_stable.zip";
+    hash = "sha256-H9Sn3I4S9CmymKIMHVagDy+7svHs285S3EJgYQo+ks0=";
+    stripRoot = false;
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/${pname} $out/bin
```

Comment:
Don't reference pname here and there, that doesn't make life better (btw `${pname}` has 8 characters and `avdump3` has 7)

Reply:
Sure. I wasn't doing that for conciseness but for flexibility in case the derivation needed to be cloned with a different name or something (I've seen this happen with a few packages in the past I think). But that's premature optimization so I'll get rid of it as you suggest.